### ISSUE TITLE
Distance duratoin

### ISSUE THREAD

**Issue by iBasit** [thephpleague/geotools#79]
Is there a way to get the distance duration like https://developers.google.com/maps/documentation/distance-matrix/intro?hl=en

```json

"duration": {
        "value": 24487,
        "text": "6 hours 48 minutes"
      }

```

**Comment by toin0u** [comment]
Hi @iBasit - it's could be interesting to have it :) But I don't know it could be implemented smoothly yet. Any ideas are welcome!

**Comment by iBasit** [comment]
I did few rough estimate and then calculate with google api for distance calculation, so far it was minute up or exact (if round it), which is not big difference.

It wont be perfect solution, but it can be initial start. 

Solution:
------------
If distance by km is not far away, then calculate 30 miles per hour (which is car speed).

If distance is far away after certain km range, then calculation should be 60 miles per hour. 

value: is seconds values
text:   is the days, hours, minutes in words. (not seconds)

**Comment by toin0u** [comment]
@iBasit Maybe we can use the API instead?

**Comment by iBasit** [comment]
Yes Google API is good for one time call on each location, but if you were building taxi app or similar app. Then checking every 30 secs for all the thousands or 100 thousands of cars will put lots of requests.  

It would not be good idea for some, but for some it will solve the issue.
